website/src/css/customTheme.css

```diff
@@ -8,10 +8,6 @@
   --ifm-color-primary-darker: #9a2521;
   --ifm-color-shadow--lw: #7f1e1b;
 }
-.navbar {
-  background: var(--ifm-color-primary);
-  color: var(--ifm-color-primary-lightest);
-}
```

Comment:
Sorry, we want the text color of the navigation bar to be #fff, like this
`.navbar a {
  color: #fff;
}`

Reply:
It is my fault for not expressing clearly, thank you for your help～

Reply:
> we would prefer the text color of the navigation bar to be #fff, like this
> 
> ```css
> .navbar a {
>   color: #fff;
> }
> ```

Yesss but here  nav bar style was already set to primary style(in docusaurus.config.js) so I thought we could remove  these lines from  css :)
.navbar {
  background: var(--ifm-color-primary);
  color: var(--ifm-color-primary-lightest);
}

Reply:
When we change the default  text color of the navigation bar to be #fff ..we also need to add hover and active selector to it IMHO :)
![Screenshot (67)](https://user-images.githubusercontent.com/53715187/105971319-64684e00-60b0-11eb-955d-b6874f6a350e.png)

Reply:
Yep, we'd better support Hover haha